src/views/DandisetLandingView/DandisetMain.vue

```diff
@@ -432,8 +487,8 @@ export default {
     },
     publishDisabled() {
       return !!(this.publishDandiset.version_validation_errors.length
-        || this.publishDandiset.asset_validation_errors.length
-        || this.publishDisabledMessage);
+          || this.publishDandiset.asset_validation_errors.length
+          || this.publishDisabledMessage);
```

Comment:
```suggestion
        || this.publishDandiset.asset_validation_errors.length
        || this.publishDisabledMessage);
```

(Try to make sure not to introduce any extraneous changes into a PR.)